.github/workflows/CI.yml

```
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - '1.8'
           - '1'
```

Maybe a good opportunity to switch to
```suggestion
          - 'min'
          - 'lts'
          - '1'
```